samples/dashboards/demo/minimal/demo.js

```diff
@@ -162,6 +187,18 @@ Dashboards.board('container', {
             },
             title: {
                 text: ''
+            },
+            tooltip: {
+                valueSuffix: ' mcg',
+                stickOnContact: true
+            },
+            accessibility: {
+                chartContainerLabel: 'Iron',
```

Comment:
- It looks like the Vitamin A and Iron is switched for the demos,
- chartContainerLabel belongs in `lang`